definition.go

```diff
@@ -1285,9 +1286,26 @@ func (gl *NonNull) Error() error {
 
 var NameRegExp, _ = regexp.Compile("^[_a-zA-Z][_a-zA-Z0-9]*$")
```

Comment:
Good point, though I'm not sure exporting this variable was deliberate. Maybe it could be marked as deprecated or simply removed anyway (current version is sub 1.0 after all).